ActionHandler/GithubHandler.php

```diff
@@ -1,16 +1,16 @@
 <?php
 
-namespace CommonGateway\PetStoreBundle\ActionHandler;
+namespace CommonGateway\GithubIntegration\ActionHandler;
```

Comment:
Whenever we need to name something, it's a good thing to keep it consistent to avoid errors. As far as I know, the repo should be renamed as well, to GithubBundle You can do this in the settings section of the repository. Feel free to ask if you can't find it.

GithubBundle*